library/data/registry-entry-attributes/src/main/java/org/quiltmc/qsl/registry/attribute/api/package-info.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * <h2>Registry Item Attributes</h2>
+ *
+ * A simple API for adding arbitrary values to any object instances managed by a {@link net.minecraft.util.registry.Registry Registry}.<p>
+ *
+ * {@link org.quiltmc.qsl.registry.attribute.api.RegistryEntryAttribute RegistryEntryAttribute} contains a number of methods for creating
+ * a new attribute.<p>
```

Comment:
"for creating a new attribute" doesn't sound right. Maybe this instead?
```suggestion
 * {@link org.quiltmc.qsl.registry.attribute.api.RegistryEntryAttribute RegistryEntryAttribute} contains a number of methods for creating
 * new attributes.<p>
```